monero-outreach-docs/translations/es/archive/getting_started_es.md

```diff
@@ -0,0 +1,197 @@
+# Empezando a Ayudar a Monero
+
+**_“La comunidad descentralizada de Monero está formada por un grupo de individuos muy dinámico y grupos propagados alrededor del planeta.”_ - 21 de agosto, 2019**
+---
+
+La comunidad descentralizada de Monero está formada por un grupo de individuos muy dinámico y grupos propagados alrededor del planeta. Un hecho poco conocido y emocionante es que Monero tiene la tercera cuenta de colaboradores más alta en todas las criptomonedas. Adicionalmente, hay muchos grupos de trabajo y proyectos que han evolucionado de la mano de Monero y es relativamente fácil empezar a ayudar.
+
+Por favor usen esto como un recurso para aprender sobre Monero, conocer sobre su ecosistema y como PUEDEN involucrarse. Bienvenidos a Monero. Esperamos que se queden.
+
+[Desarrolladores](https://www.monerooutreach.org/stories/getting-started-helping-monero.html#developers)
+[Equipos de trabajo](https://www.monerooutreach.org/stories/getting-started-helping-monero.html#workgroups)
+[Únete a la conversación e involucrate](https://www.monerooutreach.org/stories/getting-started-helping-monero.html#join)
+[Otras formas de ayudar](https://www.monerooutreach.org/stories/getting-started-helping-monero.html#otherways)
+[Monero 101](https://www.monerooutreach.org/stories/getting-started-helping-monero.html#monero101)
+[Minar Monero](https://www.monerooutreach.org/stories/getting-started-helping-monero.html#mining)
+
+### _Desarrolladores_
+
+Si eres un desarrollador, te invitamos a que te unas a algún equipo.
```

Comment:
_Si eres un desarrollador, te invitamos a que te unas **al equipo**._

Since it just says, "join the team", not "join a team" in the original file.

Reply:
Changed.